README.md

```diff
@@ -67,7 +55,7 @@ performance, small size and all the features listed below.
   * Template string support using [snabby](https://github.com/jamen/snabby).
   * Virtual DOM assertion with [snabbdom-looks-like](https://github.com/jvanbruegge/snabbdom-looks-like)
 
-## Inline example
+## Example
```

Comment:
Figured "inline" doesn't mean much here. Even less in the table of contents. So just "Example".